ckanext/protected_resources/plugin.py

```diff
@@ -142,8 +115,15 @@ def protected_resource_unlock(context, data_dict):
 
     resource = p.toolkit.get_action('resource_show')(context, {'id': data_dict['resource_id']})
 
-    if 'is_protected' in resource:
```

Comment:
10/10

Above at lines 77, 110 and 134:
```python
if user and not user.sysadmin:
    raise p.toolkit.ValidationError(["User must be a sysadmin to protect this resource"])
```
Would an anonymous user skip this check? Should it be this instead:

```python
if not user or not user.sysadmin:
```

Reply:
Nice catch!

I tested this and an anonymous user can pass the the `if user and not user.sysadmin`

I will fix this.